docs/learn/learn-cryptography.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: learn-cryptography
 title: Cryptography Explainer
-sidebar_label: Cryptography Explainer
+sidebar_label: Cryptography
```

Comment:
As they are inside the cryptography dropdown, should this be named "Explainer" instead? "Cryptography Explainer" is also okay.

Having dropdown with that same name as the title feels a little off. Cryptography > Cryptography